evergreen.yml

```diff
@@ -14,6 +14,9 @@ command_type: system
 # 12 minutes is the longest we'll ever run
 exec_timeout_secs: 3600 # 12 minutes is the longest we'll ever run
 
+ignore:
+  - "*.md"
```

Comment:
This is the only substantive change to this file. The rest were just auto formatting changes when I saved ✊